Enable dashboards in Zenkoversion

francoisferrand opened the pull request:
Issue: ZENKO-3843

francoisferrand committed:
Enable dashboards in zenkoversions

Issue: ZENKO-3843

francoisferrand committed:
Bump zenko-operator to 1.2.11

Issue: ZENKO-3843

bert-e commented:
# Hello francoisferrand,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/zenko/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue ZENKO-3843 contains:


* *None*



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `2.2.9`


Please check the `Fix Version/s` of ZENKO-3843, or the target
branch of this pull request.

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue ZENKO-3843 contains:


* *None*



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `2.2.11`


Please check the `Fix Version/s` of ZENKO-3843, or the target
branch of this pull request.

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue ZENKO-3843 contains:


* `2.2.11`



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `2.2.12`


Please check the `Fix Version/s` of ZENKO-3843, or the target
branch of this pull request.

francoisferrand committed:
Fix embedding local dashboards

Issue: ZENKO-3843

francoisferrand committed:
Do not rely on centos' rename command

This allows building Zenko ISO on other systems (ubuntu, macos...)

Issue: ZENKO-3843

lavigne958 commented on solution/build.sh:
```diff
@@ -175,7 +177,7 @@ EOF
 function get_local_dashboards()
 {
-    find ${REPOSITORY_DIR}/monitoring/ -type d -depth 1 -print0 |
+    find ${REPOSITORY_DIR}/monitoring/ -maxdepth 1 -type d -print0 |
```
what a fantastic fix ! :heart:

lavigne958 commented on solution/build.sh:
```diff
@@ -87,7 +87,9 @@ function copy_yamls()
 
     kustomize build "${zenko_operator_repo}/config/artesca-solution/crd?ref=$(zenko_operator_tag)" -o ${crd_dir}
-    rename '.yaml' '_crd.yaml' ${crd_dir}/*
+    for file in ${crd_dir}/*.yaml ; do 
+        mv $file ${file%.yaml}_crd.yaml
+    done
```
for those who don't have `rename`:  https://github.com/util-linux/util-linux

francoisferrand replied:
worth noting the binary is actually called `rename.ul` in ubuntu, though..

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers

francoisferrand committed:
Fix alerts' configuration parameters

The inputs for config options must have type "config", otherwise the
operator rejects the alert.

Issue: ZENKO-3843

francoisferrand committed:
Fix name of dashboard files

Issue: ZENKO-3843

francoisferrand committed:
Add missing `uid` field to dashboards

Zenko-operator does not generate the field if it does not exist.

Issue: ZENKO-3843

francoisferrand committed:
Bump cloudserver 8.3.8

Issue: ZENKO-3843

francoisferrand committed:
Remove 3rd party dashboards from e2e tests

The images are not pushed to any registry, and cannot thus be used for
testing at the moment.

Issue: ZENKO-3843

francoisferrand committed:
Remove Vault dashboards from e2e tests

The images is not available without registry auth.

Issue: ZENKO-3843

bert-e commented:
# Conflict




There is a conflict between your branch `feature/ZENKO-3843` and the
destination branch `development/2.2`.

Please resolve the conflict on **the feature branch** (`feature/ZENKO-3843`).

```sh
git fetch && \
git checkout origin/feature/ZENKO-3843 && \
git merge origin/development/2.2
```

Resolve merge conflicts and commit

```sh
git push origin HEAD:feature/ZENKO-3843
```

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers

francoisferrand committed:
Bump zenko-operator to 1.2.12 and backbeat to 8.3.4

Issue: ZENKO-3843

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue ZENKO-3843 contains:


* `2.2.12`



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `2.2.13`


Please check the `Fix Version/s` of ZENKO-3843, or the target
branch of this pull request.

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers

francoisferrand commented:
/approve

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/2.2`



The following branches will **NOT** be impacted:

* `development/0.9`
* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/1.3`
* `development/2.1`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/2.2`



The following branches have **NOT** changed:

* `development/0.9`
* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/1.3`
* `development/2.1`



Please check the status of the associated issue ZENKO-3843.

Goodbye francoisferrand.
